src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -251,5 +233,76 @@ canvas {
 }
 
 .asset-error {
+    font-size: 85%;
+    font-weight: 500;
     margin: auto;
 }
+
+.badge-visited, .badge-tagged {
+    color: #fff;
+    position: absolute;
+    z-index: 2;
+    right: 0.5em;
```

Comment:
right: 0; looks better to me

Reply:
`right: 0` is right up against the edge and doesn't look right to me.  Keeping with `0.5em` for now.